localization/en_GB.inc

```diff
@@ -0,0 +1,5 @@
+<?php
+$labels = array();
+$labels['internal'] = 'Mail was internal';
```

Comment:
I lack context, but Mail is the system, email is the message itself, I think here email is what's is meant, no?

Reply:
Also, I find the "was" odd here; again, I lack context, but I think Internal Mail/email could be more meaningful